production/daemonset-admin.yml

```diff
@@ -0,0 +1,31 @@
+#
```

Comment:
Not really, this is a generic role, which can be added to users, so that said user can create daemon sets. Not really sure this will be used in a real world environment, where a cluster-admin would deploy Jaeger.